uint32[] pixelCount
uint16 centreX
uint16 centreY
uint16 bestBlobSize